azure-pipelines-deploy.yml

```diff
@@ -117,6 +121,16 @@ extends:
                 parameters:
                   appName: pins-func-doc-check-doc-check-$(ENVIRONMENT)-$(REGION_SHORT)-001
                   resourceGroup: $(resourceGroup)
+          - name: Deploy Function - Deadline Submissions
+            artifact:
+              name: function-deadline-submissions
+              sourcePipeline: Back Office Build
+            condition: eq(${{ parameters.deployDeadlineSubmissionsFunction }}, 'true')
+            steps:
+              - template: ../steps/function_app_deploy.yml
+                parameters:
+                  appName: pins-func-deadline-submissions-$(ENVIRONMENT)-$(REGION_SHORT)-001
```

Comment:
based on your infra config (https://github.com/Planning-Inspectorate/infrastructure-environments/blob/main/app/components/back-office-app-services/deadline-submissions-function/function-app.tf#L5) `app_name` and `service_name` both get used, so that's why you see the others repeat.

this would be: `pins-func-deadline-submission-deadline-submission-...`